2020-12-24 17:28:40,560 INFO    MainThread:3748 [wandb_setup.py:_flush():70] setting env: {}
2020-12-24 17:28:40,561 INFO    MainThread:3748 [wandb_setup.py:_flush():70] multiprocessing start_methods=spawn
2020-12-24 17:28:40,561 INFO    MainThread:3748 [wandb_setup.py:_flush():70] setting login settings: {}
2020-12-24 17:28:40,561 INFO    MainThread:3748 [wandb_init.py:_log_setup():303] Logging user logs to C:\Users\deepu\Desktop\MyFolders\Projects\PetProjects\ActivityRecognition\src\wandb\run-20201224_172840-bgco5u5v\logs\debug.log
2020-12-24 17:28:40,561 INFO    MainThread:3748 [wandb_init.py:_log_setup():304] Logging internal logs to C:\Users\deepu\Desktop\MyFolders\Projects\PetProjects\ActivityRecognition\src\wandb\run-20201224_172840-bgco5u5v\logs\debug-internal.log
2020-12-24 17:28:42,848 INFO    MainThread:3748 [wandb_run.py:_console_start():1237] atexit reg
2020-12-24 17:28:42,849 INFO    MainThread:3748 [wandb_run.py:_redirect():1104] redirect: SettingsConsole.WRAP
2020-12-24 17:28:42,850 INFO    MainThread:3748 [wandb_run.py:_redirect():1135] Wrapping output streams.
2020-12-24 17:28:42,850 INFO    MainThread:3748 [wandb_run.py:_redirect():1151] Redirects installed.
2020-12-24 17:28:43,242 INFO    MainThread:3748 [wandb_run.py:_atexit_cleanup():1207] got exitcode: 1
2020-12-24 17:28:43,242 INFO    MainThread:3748 [wandb_run.py:_restore():1179] restore
2020-12-24 17:28:43,274 INFO    MainThread:3748 [wandb_run.py:_wait_for_finish():1324] got exit ret: uuid: "0e208fc09e6c410e8ba248896b608514"
response {
  poll_exit_response {
    file_counts {
      wandb_count: 1
    }
    pusher_stats {
      total_bytes: 794
    }
  }
}

2020-12-24 17:28:45,277 INFO    MainThread:3748 [wandb_run.py:_wait_for_finish():1324] got exit ret: uuid: "ed20193a392e4235b85e465111c546d3"
response {
  poll_exit_response {
    done: true
    exit_result {
    }
    file_counts {
      wandb_count: 5
    }
    pusher_stats {
      uploaded_bytes: 7118
      total_bytes: 7118
    }
  }
}

2020-12-24 17:28:46,414 INFO    MainThread:3748 [wandb_run.py:_show_files():1536] logging synced files
2020-12-24 17:28:46,418 INFO    MainThread:3748 [internal.py:handle_exit():138] Internal process exited
